api/.project/devstart.rb

```diff
@@ -0,0 +1,51 @@
+require_relative "common/common"
```

Comment:
Piling on: I don't mind dotfiles, but `.project` conflicts with with [a file I already have](https://help.eclipse.org/oxygen/index.jsp?topic=%2Forg.eclipse.platform.doc.isv%2Freference%2Fmisc%2Fproject_description_file.html) at this path form using some standard dev tools.